src/main/resources/assets/minecolonies/gui/townhall/windowtownhallhirecitizen.xml

```diff
@@ -0,0 +1,45 @@
+<window xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" pause="true"
+        lightbox="true"
+        xsi:noNamespaceSchemaLocation="file:../../../../java/com/minecolonies/blockout/blockOut.xsd">
+
+    <label size="200 11" pos="100 50" textalign="TOP_MIDDLE" color="white"
+           label="$(com.minecolonies.coremod.gui.townHall.hirecitizen1)"/>
+    <label size="200 11" pos="100 61" textalign="TOP_MIDDLE" color="white"
+           label="$(com.minecolonies.coremod.gui.townHall.hirecitizen2)"/>
+    <label size="200 11" pos="100 72" textalign="TOP_MIDDLE" color="white"
+           label="$(com.minecolonies.coremod.gui.townHall.hirecitizen3)"/>
+    <box size="163 100" linewidth="2" pos="127 87">
```

Comment:
probably without linewidth

Reply:
looks a bit nicer with the slightly thicker width of the boxes rectangle around the buttons I think

Reply:
better ask visuals for background though

Reply:
a 2 px width background?

Reply:
meh :D I thought about something fancy so it's in the same theme as other parts of mod